xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/docdoesnotexist.vm

```diff
@@ -29,6 +29,7 @@
   <div class="main">
   <div id="mainContentArea">
   #template("hierarchy.vm")
+  <main>
```

Comment:
inner indentation missing

Reply:
I didn't put inner indentation on some files because:
- It's already a complex PR to review, we don't need to view the 20 lines where I just added one indentation.
- The indentation policy is already loose here (cf L29-30 on this document)
- We don't lose much information. Here, the level of the L31 is weird, but except this line, it doesn't matter if it's properly indented.